api_tests/lib/comit.ts

```diff
@@ -41,7 +41,8 @@ export interface Asset {
 
 export interface Ledger {
     name: string;
-    network: string;
+    network?: string;
```

Comment:
❓ should maybe be refactored?

After quick discussion with @D4nte the network has to stay here because the `Ledger` is used for Bitcoin and Ethereum and for Bitcoin we still have a network name. 

I feel it is a bit weird to have the network name just in use for Bitcoin and the chainId just in use for Ethereum. 

❓ For Bitcoin it will always be by name?

Reply:
1. Happy to use dedicated separate interface for Bitcoin and Ethereum.
2. The requirement to change to chainId comes from the fact for each of our clients, we end-up doing a mapping to ensure that we are connected to an Ethereum node with the right network. However, for Bitcoin, we do not even do such checks.
The conclusion from the discussion with @thomaseizinger was that on Bitcoin side, we need to learn more about the usage. See how the bitcoin network is used by cnd clients. Only then, if we realise there is duplication to map the value return by cnd to something else (`testnet3`? Magic block number?) then we can do the change in cnd. If we were to change bitcoin now, it would just be a wild gues.

Reply:
Thanks for explaining @D4nte. Refactoring if needed can be in a follow-up PR.

Reply:
Already done :)